src/main/nsis/samples/conf/process-conf.xml

```diff
@@ -63,7 +63,7 @@
                 <entry key="sfdc.debugMessagesFile" value="c:\dataloader\samples\status\sfdcSoapTrace.log"/>
                 <entry key="sfdc.endpoint" value="https://login.salesforce.com"/>
```

Comment:
Is it actually necessary to allow changing the login endpoint? Seems like a needlessly risky thing to allow, as it could result in somebody sending their credentials to a third party or accidentally sending them in plain-text HTTP (actually, non-HTTPS should probably cause an error).